tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -121,9 +121,6 @@ cc-shim-v2-tarball:
 cc-virtiofsd-tarball:
 	${MAKE} $@-build
 
-cc-tdx-cloud-hypervisor-tarball:
```

Comment:
I checked that kata-containers/tests/.ci/install_cloud_hypervisor.sh never used that target, so I think we won't break anything, I hope :)